app/scripts/components/common/card/index.tsx

```diff
@@ -289,24 +284,23 @@ function CardComponent(props: CardComponentPropsType) {
     const { linkProperties: linkPropertiesProps } = props;
     linkProperties = linkPropertiesProps;
   } else {
-    const { linkTo, onLinkClick, isLinkExternal } = props;
+    const { linkTo, onClick, isLinkExternal } = props;
     linkProperties = {
       linkTo,
-      onLinkClick,
+      onClick,
```

Comment:
I consolidated `onLinkClick` to `onClick` - so this can be attached to the LinkElement directly when needed.